src/main/java/seedu/address/logic/commands/GuestAddCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+/**
+ * Adds a Guest to the address book.
+ */
+public class GuestAddCommand extends Command {
+    // below implementation more or less copied from AddCommand
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. ";
```

Comment:
Perhaps you could add the command parameters?

```suggestion
    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a guest to the address book. "
            + "Parameters: "
            + PREFIX_NAME + "NAME "
            + "[" + PREFIX_PHONE + "PHONE] "
            + "[" + PREFIX_EMAIL + "EMAIL] "
            + "[" + PREFIX_ADDRESS + "ADDRESS] "
            + "[" + PREFIX_DIETARY + "DIETARY] "
            + "[" + PREFIX_RSVP + "RSVP] "
            + "[" + PREFIX_TAG + "TAG]...\n"
            + "Example: " + COMMAND_WORD + " "
            + PREFIX_NAME + "John Doe "
            + PREFIX_PHONE + "98765432 "
            + PREFIX_EMAIL + "johnd@example.com "
            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
            + PREFIX_DIETARY + "no beef"
            + PREFIX_RSVP + "no"
            + PREFIX_TAG + "friends "
            + PREFIX_TAG + "owesMoney";
            
```

Reply:
will  be done in the next evolve!